### DISCUSSION TITLE
Convert to Ethereum

### DISCUSSION THREAD

**Discussion by FolderHyph** [DeFiCh/app#782] [Q&A] [answered]
Hello,

I just wanted to understand if now it's possible to convert DFI into Ethereum within the wallet and then send the ethereum to an ethereum wallet.

**Comment by MRHMuc** [comment, marked as answer]
Hi @FolderHyph 

Currently the best way would be to convert your DFI in wrapped ETH in the wallet - next, send it to CAKE - next, send it from there to your ethereum wallet. Other option is to send your DFI to one of the exchanges (e.g. Bittrex or Kucoin) and go from there. 

In the near future ATOMIC SWAP will be available. This will enable to transact BTC in and out of DeFiChain without going through CAKE.

Best, Mark
